### PR TITLE
Update style-guide.md

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Added kubectl and Minikube to the Kubernetes.io word list. Please review.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.10 Features: set Milestone to 1.10 and Base Branch to release-1.10
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> NOTE: After opening the PR, please *un-check and re-check* the ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) box so that maintainers can work on your patch and speed up the review process. This is a temporary workaround to address a known issue with GitHub.> 
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

### CODE REVIEW

**Comment by heckj** [comment]
I'm good with those! 👍

**Comment by Rajakavitha1** [comment]
@heck  I also added Persistent Volumes and Salt to the list and set the table in an alphabetical order. Please review.

File: docs/home/contribute/style-guide.md [hunk 1]
```diff
@@ -179,8 +179,12 @@ A list of Kubernetes-specific terms and words to be used consistently across the
 
 <table>
   <tr><th>Term</th><th>Usage</th></tr>
-  <tr><td>Kubernetes</td><td>Kubernetes should always be capitalized.</td></tr>
   <tr><td>Docker</td><td>Docker should always be capitalized.</td></tr>
+  <tr><td>Kubernetes</td><td>Kubernetes should always be capitalized.</td></tr>
+  <tr><td>kubectl</td><td>kubectl should always not be capitalized.</td></tr>
+  <tr><td>Minikube</td><td>Minikube should always be capitalized.</td></tr>
+  <tr><td>Persistent Volumes</td><td>Persistent Volumes should always be capitalized.</td></tr>
+  <tr><td>Salt</td><td>Salt should always be capitalized.</td></tr>
```
Comment (heckj) [inline docs/home/contribute/style-guide.md]: is there still much Salt referenced? It seems to be decreasing (a large section was deprecated related to it a few months ago) - this worth a call out?

**Comment by Bradamant3** [comment]
I'd like to see us discuss more carefully the capitalization issue wrt objects like PersistentVolume, where the object name is Pascal Case but where the words can also refer to a more generic thing -- persistent volume here. Consistency and clarity might suggest that we should always refer to objects as something like "PersistentVolume object/s" -- because pluralizing the object name is programmatically inaccurate. (Same holds for other object names where the name is also a common word, for example node or deployment.)
But ... compliance with such persnickitiness is never gonna happen, and is probably not a good idea anyway. 
Recommend dropping "Persistent Volume" from the list until we can discuss the larger issue.

**Comment by chenopis** [comment]
I'm good w/ these. Yes, we should have a broader discussion about capitalization, etc.

/lgtm
/approve

**Pull request merged**